rocky-backend/DBtest.py

```
@@ -0,0 +1,95 @@
+
+
+# crete course
+res = requests.post(f"{BASE}/courses", json={
+    "name": "CS101",
+    "semester": {"year": 2026, "term": "Spring"}
+})
+test("Create valid course", res.status_code == 200)
+
+
+# create invalid course
+res = requests.post(f"{BASE}/courses", json={
```

Ruff (0.15.9)

[error] Probable use of `requests` call without timeout

(S113)